ar4si.go

```diff
@@ -69,32 +87,48 @@ func (o *TrustTier) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// AttestationResult represents the result of an evidence appraisal by the
+// verifier.  It wraps the AR4SI trustworthiness vector together with other
+// metadata that are relevant to establish the appraisal context - the evidence
+// itself, the appraisal policy used, the time of appraisal.
+// The AttestationResult is serialized to JSON and signed by the verifier using
+// JWT.
 type AttestationResult struct {
 	Status            *TrustTier   `json:"status"`
+	Profile           *string      `json:"eat_profile"`
```

Comment:
What is the purpose of adding profile?

I thought the AttestationResult Object should be profile Agnostic, so that any RP can assess the Verification Results coming from any Verifier?

Reply:
> What is the purpose of adding profile?

to make it an EAT

> I thought the AttestationResult Object should be profile Agnostic, so that any RP can assess the Verification Results coming from any Verifier?

I am not sure I understand what you mean by "profile agnostic"